pyaedt/icepak.py

```diff
@@ -3313,50 +3320,27 @@ def assign_stationary_wall(
             or ``"Heat Transfer Coefficient"``.
         name : str, optional
             Name of the boundary condition. The default is ``None``.
-        temperature : str or float or dict, optional
+        temperature : str or float or dict or BoundaryDictionary, optional
             Temperature to assign to the wall. This parameter is relevant if
             ``ext_condition="Temperature"``. If a float value is specified, the
-            unit is degrees Celsius. A dictionary can be used for transient
-             assignment. The dictionary should contain three keys: ``"Type"``, ``"Function"``, and
-             ``"Values"``.
-
-             - The value for the ``"Type"`` key must be ``"Transient"``.
-             - Accepted values for the ``"Function"`` key are: ``"Linear"``, ``"Power Law"``, ``"Exponential"``,
-             ``"Sinusoidal"``, ``"Square Wave"``, and ``"Piecewise Linear"``.
-             The ``"Values"`` key contains a list of strings containing the parameters
-            required by the ``"Function"`` key selection. For example, ``"Linear"`` requires two parameters:
-            the value of the variable at t=0 and the slope of the line. The parameters required by
-            each ``Function`` key selection is in Icepak documentation. The parameters must contain the
-            units where needed. The default is ``"0cel"``.
-        heat_flux : str or float or dict, optional
+            unit is degrees Celsius. Assign a transient condition using the
+            result of a function with the pattern `create_*_transient_assignment`.
```

Comment:
```suggestion
            result of a function with the pattern ``create_*_transient_assignment``.
```
A single backtick only italicizes, while a double backtick treats it as a conde entity.